packages/api/src/user.js

```diff
@@ -172,3 +177,28 @@ export async function userTokensPost (request, env) {
 
   return new JSONResponse()
 }
+
+/**
+ * Retrieve user auth tokens.
+ *
+ * @param {AuthenticatedRequest} request
+ * @param {import('./env').Env} env
+ */
+export async function userTokensGet (request, env) {
+  const res = await env.db.query(gql`
+    mutation FindAuthTokensByUser($user: ID!) {
```

Comment:
Why is `FindAuthTokensByUser` a `mutation`? Sounds like a `query`

Reply:
it does